DESCRIPTION

```diff
@@ -31,27 +31,34 @@ Depends:
     R (>= 3.1),
     ggplot2 (>= 3.3.0)
 Imports:
+    dplyr,
```

Comment:
Things happened in 1.0.0. Good to be safe. (Even though `.data` was added in 0.7.0)

```suggestion
    dplyr (>= 1.0.0),
```